### PR TITLE
Added "[sheet" to the texture special commands.

### PR DESCRIPTION
**Pull request by lupuchard**
"[sheet:WxH:X,Y" assumes the base image is a tilesheet with W*H tiles on it and crops to the tile at position X,Y. It's like "[verticalframe" but in 2d.
For testing, I combined the four default_chest images into one.

### CODE REVIEW

**Comment by paramat** [comment]
Any more reviews before we ask for another rebase?

File: doc/lua_api.txt [hunk 1]
```diff
@@ -262,10 +262,19 @@ Advanced texture modifiers:
...
+    Retrieves a tile at position x,y from the base image
+    which it assumes to be a tilesheet with dimensions w,h.
+
   [colorize:<color>
     Colorize the textures with given color
     <color> as ColorString
 
+  [sheet:<w>x<h>:<x>,<y>
+    Retrieves a tile at position x,y from the base image
+    which it assumes to be a tilesheet with dimensions w,h.
```
Comment (sfan5) [inline doc/lua_api.txt]: duplicate

**Comment by paramat** [comment]
The commits are a mess, please can you squash?

**Comment by sofar** [comment]
replaced by #4861 (rebased and re-tested version of this pr).

**Pull request closed without merging**